src/pages/vacinasPaciente/vaccine/Vaccine.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+
+import arrowUp from '../../../assets/arrow-up.svg';
+import arrowDown from '../../../assets/arrow-down.svg';
+import { VaccineType } from "../../../Types/VaccineTypes";
+
+import { VaccineButton, VaccineContainer, VaccineDate, VaccineDescription, VaccineText, VaccineTitle } from "./Vaccine.style";
+
+function Vaccine({ title, observation, scheduledDate, status, applicationDate }: VaccineType) {
+    const [showDescription, setShowDescription] = useState(false);
+
+    function toggledescription() {
+        setShowDescription(!showDescription)
+    }
+
+    function getVaccineStatus() {
+        switch (status) {
+            case 'applied':
+                return 'applied'
+            case 'not applied':
+                return 'not-applied'
+            case 'late':
+                return 'late'
+            case 'schedule':
+                return 'schedule'
+            default:
+                return ''
+        }
+    }
```

Comment:
Ficou muito bom, só queria te apresentar essa outra forma de criar um switch case
https://ultimatecourses.com/blog/deprecating-the-switch-statement-for-object-literals

Reply:
Legal